### DISCUSSION TITLE
Nuova istanza Home Assistant sulla stessa rete

### DISCUSSION THREAD

**Discussion by Stefano201** [indomus/forum#2901] [HUB: Home Assistant]
Buonasera, a causa di alcuni problemi alla mia istanza principale di HA, ho deciso di crearne un'altra e piano piano personalizzarla di nuovo seguendo ciò che avevo già fatto sulla principale.
Avendo un altro RPi4 e un'altra sd ho proceduto con il Raspberry Pi Manager (dove ho impostato anche le credenziali wifi). Tutto ok fino a quando non inserisco la micro sd nella macchina. Il raspberry si connette alla rete, vedo anche l'indirizzo ip (dalla pagina del modem) ma non riesco in alcun modo a raggiungere la pagine http://192.168.1.32:8123 

Non credo che il problema sia il fatto che sul modem la porta 8123 è aperta per l'altra istanza HA...sbaglio?

**Comment by snakuzzo** [comment]
Però perdonami...uno cosa dovrebbe risponderti ?
Se non dai nemmeno un minimo di dettaglio di che problema hai a raggiungere quell'IP su quella porta.
Non dici se ti torna un errore HTTP, un timeout, qualcosa...niente...così come hai scritto potrebbe essere tutto o niente.

Poi cosa c'entra il port forwarding ? Quello serve a raggiungere HA dall'esterno, se il problema ce l'hai all'interno della tua LAN quello non influisce in alcun modo (per favore chiarisciti questo concetto perché veramente è una cosa che non c'entra nulla)

P.S. Ti ho risposto pure sull'altro topic, a sto punto m'hai fatto solo perdere tempo :D

**Reply by Stefano201** [reply]
Non avendo avuto nessun riscontro da ieri, stamattina ho deciso di procedere il altro modo. L'ho letta e ti ringrazio, ma al momento non mi trovo una tastiera usb da collegare e quindi devo desistere.

Per il port forwarding infatti ho già detto che non credo sia quello il problema, e ho chiesto solo conferma.

Per la questione della pagina web questo è quello che riscontro:

<img width="778" alt="Screenshot 2024-02-09 alle 15 42 37" src="https://github.com/indomus/forum/assets/159457620/773be031-62ee-43a7-8e9a-ec7d0b448e6b">

**Reply by Stefano201** [reply]
Lato router

<img width="366" alt="Screenshot 2024-02-09 alle 15 44 14" src="https://github.com/indomus/forum/assets/159457620/d63cf9fe-5167-4f4b-a4c6-003499010968">

**Reply by snakuzzo** [reply]
Se ti dà connection refused, non c'è alcun servizio in ascolto su quella porta.
Collega un monitor, riavvia...e vedi cosa accade

**Reply by Stefano201** [reply]
Chiedo venia...ho installato sulla sd Raspberry Pi Desktop anziché HaOS..imperdonabile lo so 🤦🏼‍♂️
